website/docs/components/code-block/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,31 @@
+## Anatomy
+
+![Anatomy of codeBlock](/assets/components/code-block/code-block-anatomy.png)
+
+| Element          | Usage    |
+|------------------|----------|
+| Title            | Optional |
+| Description      | Optional |
+| Line numbers     | Optional |
+| Copy button      | Optional |
+| Code snippet     | Required |
+| Highlighted line | Optional |
+
+## Syntax highlighting
+
+To aid in understanding how the highlighting theme is applied via Prism's tokens, we've provided a high-level, non-exhaustive list of token names and how they might be applied depending on the syntax.
+
+| Color | Usage |
+|-------|-------|
+| Cyan | Property, url, or operator |
+| Blue | Function, builtins |
+| Orange | Strings, characters |
+| Purple | Booleans, numbers |
+| Green | Keywords, class names, saving the world |
+| Red | Important items |
+| White | Default color within the code block, also used for punctuation (`<`, `{ }`, `=`, etc) |
+| Gray | Used for comments across languages |
```

Comment:
[suggestion] Can a little color swatch be added to each of these? It may be more visually impactful, but if it's too complex, it's not necessary.

Reply:
I tried to add a color swatch from the docs components but had a bit of a problem because the properties are de-structured within the component in a way that I'm not familiar with. Though, adding an image may suffice? Let me try that.

Reply:
@jorytindall I can help with it, we can re-use the color preview of the tokens: http://localhost:3001/foundations/tokens

Reply:
Thanks to @didoo, there are some simple swatches in each row corresponding with the syntax color now! 🎉